openmetadata-ui/src/main/resources/ui/src/components/DataQuality/TestSuite/TestSuiteList/TestSuites.component.tsx

```diff
@@ -164,9 +164,9 @@ export const TestSuites = ({ summaryPanel }: { summaryPanel: ReactNode }) => {
         title: `${t('label.success')} %`,
         dataIndex: 'summary',
         key: 'success',
-        render: (value: TestSummary) => {
+        render: (value: TestSuite['summary']) => {
           const percent =
-            value.total && value.success ? value.success / value.total : 0;
+            value?.total && value?.success ? value.success / value.total : 0;
```

Comment:
@ShaileshParmar11, if both `total` and `success` are zero, the division will result in `NaN`. Could we add a safeguard to handle this?